docs/src/routes/docs/_layout.svelte

```diff
@@ -132,6 +132,10 @@
         },
       ],
     },
+    {
+      title: 'Utilities',
+      segment: 'utils',
```

Comment:
A mismatch between "utilities" in the left nav and "utils" in the breadcrumbs

Reply:
Because the breadcrumbs component uses the `segment` and not the `title`. Should I change it to use the title instead or do you want the segment (and therefore the URL as well) to use the full word?

Reply:
Use the full word for the URL